common/src/java/org/apache/hadoop/hive/common/cli/ShellCmdExecutor.java

```diff
@@ -22,12 +22,14 @@
 import java.io.PrintStream;
 
 import org.apache.hive.common.util.StreamPrinter;
+import org.apache.hive.common.util.SuppressFBWarnings;
 
 public class ShellCmdExecutor {
   private String cmd;
   private PrintStream out;
   private PrintStream err;
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP2", justification = "intended_to_do")
```

Comment:
Does `@SuppressFBWarnings`  have any help about upgrading the jdk17?

Reply:
@zhangbutao I have added the reasoning behind this change in the comment above: [https://github.com/apache/hive/pull/5404#discussion_r1736025764](https://github.com/apache/hive/pull/5404#discussion_r1736025764) . Please check that out. Thanks.